src/lib/rfc3339_time/rfc3339_time.ml

```diff
@@ -0,0 +1,10 @@
+open Core_kernel
+
+let get_rfc3339_time () =
```

Comment:
Since there's a similar function in `Uptime_service`, can you use this new library function there, so the code isn't duplicated?